packages/muelu/doc/UsersGuide/masterList.xml

```diff
@@ -892,6 +892,14 @@
     </parameter>
 
     <parameter>
+      <name>repartition: target rows per proc</name>
```

Comment:
I prefer "target rows per process".

Reply:
Shouldn't we then change "min rows per proc" to "min rows per process" too?

Reply:
Shoot, you're right, and if we change "min rows per proc", that will cause all sorts of problems. So it's easiest to keep it as you have it: "target rows per proc".